content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -0,0 +1,609 @@
+We've shown how a `ClusterPolicy` doesn't have to "enforce" settings, rather it can be used to set and extend the default options. 
+`Certificate` users can reduce their YAML, whilst maintaining the flexibility to override any value when needed. 
+
+We have shown how a few simple policies can change the user experience creating `Certificate` resources from:
+
+```yaml file=../../../../public/docs/tutorials/certificate-defaults/cert-test-revision-override.yaml
+```
+🔗 <a href="cert-test-revision-override.yaml">`cert-test-revision-override.yaml`</a>
+
+To instead only need to specify the configuration important to them, for example:
+
+```yaml file=../../../../public/docs/tutorials/certificate-defaults/cert-test-minimal.yaml
+```
+🔗 <a href="cert-test-minimal.yaml">`cert-test-minimal.yaml`</a>
+
+With these policies we achieved our objective and have enabled users to submit minimal `Certifiate` resources, with only a single field contained within the specification, the `dnsNames` entry.
+Every other specified field was automatically defaulted using Kyverno with `ClusterPolicy` which would typically be setup by a platform administrator.
+
+You may have noticed that in the second Kyverno `ClusterPolicy`, we actually used some `Certificate` resource metadata to create the `secretName` field.
+You read more about this on the [Kyverno documentation](https://kyverno.io/docs/writing-policies/variables/#variables-from-admission-review-requests). 
+
+# Cleanup
+
+If you created the kind cluster for this tutorial you can simply run:
+
+```shell
+kind delete cluster --name defaults
+```
+
+Otherwise to remove all resources deployed in this tutorial:
+
+```shell
+# Assuming you are running from this directly or saved all the files to yamls/
+kubectl delete -f yamls/
+helm uninstall kyverno -n kyverno-system
+helm uninstall cert-manager -n cert-manager
+helm uninstall ingress-nginx -n ingress-nginx
+```
+
+# Appendix
+
+## cert-manager version requirement
+
+Prior to cert-manager version v1.14.X, cert-manager's `MutatingWebhookConfiguration` was trigger by all cert-manager.io resources including `Certificates`.
+In reality this webhook is only in place to affect `CertificateRequest` resources, but had an unintended consequence that meant Kyverno policies as we had written would not operate as intended.
+
+When a `Certificate` resource is applied to a Kubernetes cluster, mutating webhooks are applied before validating webhooks. 
+When the existing cert-manager `MutatingWebhookConfiguration` runs it will add the required field with an empty value, such as: `secretName: ""`, if there is no value in a required field. 
+The consequence of this action is that our Kyverno policies will not apply as an empty value is already present.
+
+Starting at v1.14.X the cert-manager-webhook `MutatingWebhookConfiguration` resource has been scoped back to only affect `CertificateRequest` resource. 
```

Comment:
```suggestion
Starting with v1.14.x, the cert-manager-webhook `MutatingWebhookConfiguration` resource has been scoped to only affect `CertificateRequest` resources.
```

Reply:
I should have called it out earlier, I prefer lowercase `.x` throughout.

Reply:
I just thought, should I not just use `v1.14` rather than `v1.14.x`?